model_parameters/default_dashboard.yml

```diff
@@ -3,16 +3,14 @@ dashboard: # Parameters for the dashboard (visualisation), all are optional
   output_name: test_dashboard # name for dashboard output file (.html will be added)
   data_dir: devon_data # Root directory to load data from
   start_day: 0    # Start day
-  end_day: 50      # End day (inclusive)
+  end_day: 99      # End day (inclusive)
   start_run: 0   # Iteration/run to start
   end_run: 10     # Last iteration/run to include
   # define scenarios (first will be used as control), any number
   scenario_name: # names for scenario
   - Control
-  - Alternative
   scenario_dir: # directory for scenario (subdir of /output), same order as names above
-  - default
-  - scenario1
+  - TestScenario
   # for all scenarios
```

Comment:
Does changing this but not default.yml for the model lead to any problems when then loading the python dashboard? (I can't quite remember how these two files interact) but obviously want the scenario_dir folder to line up with scenario entry in `defaults.yml`

Reply:
ahh crap I didn't mean to change this file, the OpenCL dashboard doesn't even use this file, so it should be totally separate. I just changed this locally to show results from the python model. I'll revert this change!